src/types/detailTypes.ts

```diff
@@ -0,0 +1,27 @@
+import { ContentType } from './searchTypes';
```

Comment:
Moet deze niet in de `@viaa/avo2-types`-package komen?

Reply:
yep, eens hij beschikbaar is